docs/backend/guides/open-telemetry.md

```diff
@@ -0,0 +1,251 @@
+ecs.task.memory.usage.max       container.memory.usage.max    Bytes
+ecs.task.memory.usage.limit     container.memory.usage.limit  Bytes
+ecs.task.memory.reserved        container.memory.reserved     Megabytes
+ecs.task.memory.utilized        container.memory.utilized     Megabytes
+		
+ecs.task.cpu.usage.total        container.cpu.usage.total	    Nanoseconds
+ecs.task.cpu.usage.kernelmode   container.cpu.usage.kernelmode	Nanoseconds
+ecs.task.cpu.usage.usermode     container.cpu.usage.usermode	Nanoseconds
+ecs.task.cpu.usage.system       container.cpu.usage.system      Nanoseconds
+ecs.task.cpu.usage.vcpu         container.cpu.usage.vcpu        vCPU
+ecs.task.cpu.cores              container.cpu.cores             Count
+ecs.task.cpu.onlines            container.cpu.onlines           Count
+ecs.task.cpu.reserved           container.cpu.reserved          vCPU
+ecs.task.cpu.utilized           container.cpu.utilized          Percent
+		
+ecs.task.network.rate.rx	          container.network.rate.rx	            Bytes/Second
+ecs.task.network.rate.tx	          container.network.rate.tx	            Bytes/Second
+ecs.task.network.io.usage.rx_bytes	  container.network.io.usage.rx_bytes	Bytes
+ecs.task.network.io.usage.rx_packets  container.network.io.usage.rx_packets	Count
+ecs.task.network.io.usage.rx_errors	  container.network.io.usage.rx_errors	Count
+ecs.task.network.io.usage.rx_dropped  container.network.io.usage.rx_dropped	Count
+ecs.task.network.io.usage.tx_bytes	  container.network.io.usage.tx_bytes	Bytes
+ecs.task.network.io.usage.tx_packets  container.network.io.usage.tx_packets	Count
+ecs.task.network.io.usage.tx_errors	  container.network.io.usage.tx_errors	Count
+ecs.task.network.io.usage.tx_dropped  container.network.io.usage.tx_dropped	Count
+		
+ecs.task.storage.read_bytes	  container.storage.read_bytes	Bytes
+ecs.task.storage.write_bytes  container.storage.write_bytes	Bytes
+```
+
+#### Elasticache (Redis)
+
+The Milmove app uses a Redis cache to store session data for auth purposes.  I omitted some of the credit balance metrics that infra uses but the full documentation is available here [https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/CacheMetrics.Redis.html](https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/CacheMetrics.Redis.html).
+
+- ActiveDefragHits
+- BytesUsedForCache
+- CacheHitRate
+- CacheHits
+- CacheMisses
+- CPUUtilization
+- CurrConnections
+- CurrItems
+- DatabaseMemoryUsagePercentage
+- DBOAveerageTTL
+- EngineCPUUtilization
+- Evictions
+- FreeableMemory
+- GetTypeCmds
+- GetTypeCmdsLatency
+- IsMaster
+- KeyBasedCmds
+- KeyBasedCmdsLatency
+- MasterLinkHealthStatus
+- MemoryFragmentationRatio
+- NetworkBytesIn
+- NetworkBytesOut
+- NetworkPacketsin
+- NetworkPacketsout
+- NewConnections
+- Reclaimed
+- ReplicationBytes
+- ReplicationLag
+- SaveInProgress
+- SetTypeCmds
+- SetTypeCmdsLatency
+- StringBasedCmds
+- StringBasedCmdsLatency
+- SwapUsage
+
+### Resources
```

Comment:
Can you link to OTel official docs here? And/or some of the other existing documentation that was linked to above

Reply:
Added some general links